api/python/cell_census/src/cell_census/_experiment.py

```diff
@@ -1,33 +1,42 @@
+# Copyright (c) 2022-2023 Chan Zuckerberg Initiative
+#
+# Licensed under the MIT License.
+
+"""
+Experiments handler.
+
+Contains methods to retrieve SOMA Experiments.
+"""
+
 import re
 
 import tiledbsoma as soma
 
 
 def _get_experiment(census: soma.Collection, organism: str) -> soma.Experiment:
-    """
-    Given a census ``soma.Collection``, return the experiment for the named organism.
+    """Given a census ``soma.Collection``, return the experiment for the named organism.
     Organism matching is somewhat flexible, attempting to map from human-friendly
-    names to the underlying collection element name.  Will raise a ``ValueError`` if
-    unable to find the specified organism [lifecycle: experimental].
+    names to the underlying collection element name.
+
+    [lifecycle: experimental]
+
+    Args:
+        census: soma.Collection, The census
```

Comment:
Suggestion only - I think the args with a linebreak reads better in pydoc and in an IDE.  It is explicitly allowed per the Google style guide - the linebreak goes just before the description of the arg.

In other words:

```
    Args:
        census: soma.Collection
            The census
        organism: str
            The organism name, eg., ``Homo sapiens``
```

Reply:
I think it's a good idea, especially since some of the later methods have a lot of arguments. I will change it.